mls-rs/Cargo.toml

```diff
@@ -111,7 +111,7 @@ required-features = ["x509"]
 
 [[example]]
 name = "large_group"
-required-features = []
+required-features = ["test_util"]
```

Comment:
That's a little odd to have to enable a test-related feature for an example. I don't remember why `DefaultMlsRules` is defined differently in test vs no-test.

Reply:
Agreed we shouldn't require test_util for that

Reply:
I made the customizable ones default, since it's handy and doesn't hurt.